Content.Shared/Humanoid/Markings/MarkingColoring.cs

```diff
@@ -0,0 +1,232 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Humanoid.Markings;
+
+[Serializable, NetSerializable]
+public enum MarkingColoringType : byte
+{
+    /// <summary>
+    ///     Color applied from "color" property
+    /// </summary>
+    SimpleColor,
+
+    /// <summary>
+    ///     Color applied from humanoid skin
+    /// </summary>
+    SkinColor,
+
+    /// <summary>
+    ///     Color applied from humanoid hair color
+    /// </summary>
+    HairColor,
+    
+    /// <summary>
+    ///     Color applied from humanoid facial hair color
+    /// </summary>
+    FacialHairColor,
+
+    /// <summary>
+    ///     Color applied from humanoid hair or facial hair color
+    /// </summary>
+    AnyHairColor,
+
+    /// <summary>
+    ///     Color applied from skin, but much darker.
+    /// </summary>
+    Tattoo,
+
+    /// <summary>
+    ///     Color applied from humanoid eye color
+    /// </summary>
+    EyeColor
+}
+
+
+[DataDefinition]
+[Serializable, NetSerializable]
+public sealed class MarkingColors
```

Comment:
this could use doccomments explaining what the default coloring stuff is intended to do